overwatcheat.cfg

```diff
@@ -1,15 +1,27 @@
-# The virtual key code key which activates the aim bot when held
-aim_key=1
+# The virtual key code key which activates the aim bot when held.
+# When using flicking mode, do not set this to left mouse button.
+aim_key=6
+
+# 0 for tracking(e.g. soldier, tracer), 1 for flicking(e.g. mcreee, ashe)
+aim_mode=1
+
+# Maximum pixel distance to target to shoot when flicking. 
+# Setting this too high may cause it to shoot prematurely and miss,
+# setting this too low may cause it to never shoot.
+flick_shoot_pixels=5
+
+# Time off between flick shots (200 for mccree, 350 for ashe, 270 for widow)
+flick_pause_duration=200
 
 # Your in-game sensitivity.
-sensitivity=15.0
+sensitivity=3.0
```

Comment:
@Tailen just as a nitpick, I suggest to not change the values that do not belong to the functionality that you aimed to change. Although that makes it easier to commit, ngl.